platformio_esp82xx_envs.ini

```diff
@@ -194,6 +194,11 @@ platform_packages         = ${regular_platform.platform_packages}
 build_flags               = ${regular_platform.build_flags}
                             ${esp8285_1M.build_flags}
 
+[env:Peter_Custom_ESP8285_1M]
```

Comment:
Why include a build name with your name in it?
If the Custom build is missing for ESP8285, then please include it, but without your name.